Team9ProjectTests/src/com/indragie/comput301as1/test/ReceiptTests.java

```diff
@@ -0,0 +1,17 @@
+package com.indragie.comput301as1.test;
+
+import com.indragie.cmput301as1.ExpenseItemAddActivity;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+public class ReceiptTests extends ActivityInstrumentationTestCase2<ExpenseItemAddActivity> {
+
+	public ReceiptTests(String name) {
```

Comment:
Are there supposed to be tests here? I don't know how much of this stuff is testable but I assumed there might be since this class was committed.